services/common/boltzmann/response.js

```diff
@@ -75,18 +77,32 @@ function error(err, status = 500, extraHeaders = {}) {
   if (typeof err === 'string') {
     err = { message: err, code: 'ENOTSUPPLIED' };
   }
+
+  if (isDev()) {
+    err.trace = new Error().stack;
+  }
+
   const r = new Response(JSON.stringify(err), { status, headers });
   return r;
 }
 
+error.coded = coded;
+function coded(code, ...args) {
+  return error(Object.assign(new Error(code), { code }), ...args);
+}
+
+function empty(status = 204, headers = {}) {
+  return new Response('', { status, headers });
+}
+
 function authneeded(message, status = 401, extraHeaders = {}) {
   const headers = new Headers({
     'www-authenticate': 'bearer',
     'content-type': 'application/json',
     ...extraHeaders
   });
   if (typeof message === 'string') {
-    message = { error: message };
+    message = { message, code: 'authneeded' };
```

Comment:
Standardize a bit on `message` (which is used in `response.error`), while we're at it.